src/main/java/nwoolcan/view/model/warehouse/ViewQueriedArticle.java

```diff
@@ -0,0 +1,29 @@
+package nwoolcan.view.model.warehouse;
+
+import nwoolcan.view.model.warehouse.article.ViewArticle;
+
+import java.util.List;
+
+/**
+ * Model of the view of the list of the articles.
+ */
+public final class ViewQueriedArticle {
```

Comment:
usual convetion for view model classes is NameToRepresentViewModel

Reply:
and for details, NameToRepresentDetailViewModel

Reply:
l'uscio e' aperto